website/settings/defaults.py

```diff
@@ -580,6 +580,10 @@ class CeleryConfig:
                 'schedule': crontab(minute=0, hour=9),  # Daily 4:00 a.m.
                 'kwargs': {'yesterday': True}
             },
+            'data_storage_usage': {
+              'task': 'management.commands.data_storage_usage',
+              'schedule': crontab(day_of_month=1, minute=30, hour=9),  # First of the month at 4:30 a.m.
```

Comment:
Probably, but I'm not sure. This will probably have to be re-negotiated with Matt. That being said, the keen_events script only grabs nodulous and users, and shouldn't overlap at all with this script. And it's not processor-intensive, so I'm hoping this is a good place to put it.